Packs/Jira/ReleaseNotes/3_0_12.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Atlassian Jira v2
+
+Added support for additional arguments in ***jira-get-organizations***.
+    - projectKey: The ID of the service desk from which the organization list will be returned. This can alternatively be a project identifier.
+    - start: The starting index of the returned objects. Base index 0. See the Pagination section for more details.
+    - limit: The maximum number of items to return per page. Default, 50. See the Pagination section for more details.
+    - accountId: The account ID of the user, which uniquely identifies the user across all Atlassian products. For example, 5b10ac8d82e05b22cc7d4ef5
```

Comment:
```suggestion
    - *projectKey*: The ID of the service desk from which the organization list will be returned. This can alternatively be a project identifier.
    - *start*: The starting index of the returned objects. Base index 0. See the Pagination section for more details.
    - *limit*: The maximum number of items to return per page. Default, 50. See the Pagination section for more details.
    - *accountId*: The account ID of the user, which uniquely identifies the user across all Atlassian products. For example, 5b10ac8d82e05b22cc7d4ef5.
```